### DISCUSSION TITLE
SFRM72-FU-DL  Help

### DISCUSSION THREAD

**Discussion by Holydol** [keirf/flashfloppy#790] [General]
Hi
I recently purchased a Gotek SFRM72-FU-DL  for my w30 Roland sampler but I upgrade the firmware to 3.39 and now in the display it shows F-F.  before it was 000   when no flash disk was inserted.  how can i bring it back to 000?

Thank you

**Comment by keirf** [comment]
You have overwritten the factory firmware with FlashFloppy firmware. They do not behave the same and you cannot revert to factory firmware once it is overwritten. FlashFloppy currently only works with IMG disk files so to use this firmware you will need a FAT32 formatted usb stick containing (probably) 1.44mb IMG files.

**Comment by Holydol** [comment]
Thank you for the information. So there is no way to do it right  even with slashing it?

**Comment by keirf** [comment]
That's right. Because noone has a copy of the original factory firmware because it is read protected. Turning off the read protection mass erases the device.
